src/Tests/Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="Particular.Approvals" Version="0.2.0" />
+    <PackageReference Include="Particular.CodeRules" Version="0.4.0" PrivateAssets="All" />
```

Comment:
Nitpick: Test project doesn't need this to be `PrivateAssets="All"`

Reply:
Good point. I'll change that.